docs/connectors/slack.md

```diff
@@ -74,15 +76,8 @@ You need to choose between two backends. The [Events API](https://api.slack.com/
 
 If you are unsure which one is the best for you, [Slack Faq](https://api.slack.com/faq#events_api) provide differences between those two.
 
-```eval_rst
-.. note::
-  You should follow the instructions for the Event API first when configuring your slack connector, even if you are planning
-  on using slack in Socket Mode.
-```
```

Comment:
Why was this removed?

Reply:
Event API does not need to be used first. You can use Socket Mode even without the Event API. Generally, the documentation related to the connection with slack is a little bit unclear, that's why this pr.

Reply:
> Event API does not need to be used first.

We generally recommend everyone set up the event API because it is more fully featured than the Socket API. That's why this message is here.

Reply:
I understand the situation. However, the current note stating that the Event API is a prerequisite even for Socket Mode may not be entirely accurate. Would it be acceptable if I modified the note to recommend using the Event API for its full features?

Reply:
Yep that sounds great

Reply:
ok! I'll come back later with the changes. Thanks